docs/contributing.rst

```diff
@@ -17,10 +17,18 @@ In particular, we need help in the following areas:
 * Responding to questions from other users on `StackExchange
   <https://ethereum.stackexchange.com>`_ and the `Viper Gitter
   <https://gitter.im/ethereum/viper>`_
+* Suggesting Improvements
+<https://github.com/ethereum/viper/tree/master/.github/VIP_TEMPLATE.md`_
```

Comment:
@DavidKnott Thanks for adding it to the docs! There is a closing ">" missing before the "`" here and in the change below.